.ci.yaml

```diff
@@ -306,7 +306,7 @@ targets:
     recipe: packages/packages
     timeout: 60
     properties:
-      target_file: android_platform_tests.yaml
+      target_file: android_platform_tests_api_33.yaml
```

Comment:
Can you add a comment that says all the shards have to have the exact same dependency list since they are not actually independent?